### PR TITLE
Fix voice recording view layout and colors

### PR DESCRIPTION
**Pull request by laevandus**
### 🔗 Issue Link

[IOS-604](https://linear.app/stream/issue/IOS-604/)

### 🎯 Goal

Message of the voice recording should be outside of the recording view. Background colors were off as well.

### 🧪 Testing

N/A

### 🎨 Changes

| Before  | After |
| ------------- | ------------- |
| before-light-2  | after-light-2 |
| before-dark-2  | after-dark-2  |

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] Changelog is updated with client-facing changes
- [x] New code is covered by unit tests
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created)

### CODE REVIEW

File: Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift [hunk 1]
```diff
@@ -40,32 +40,38 @@ public struct VoiceRecordingContainerView<Factory: ViewFactory>: View {
...
+                }
             }
+            .padding(.all, 8)
+            .background(Color(colors.background8))
+            .cornerRadius(16)
             if !message.text.isEmpty {
                 AttachmentTextView(message: message)
                     .frame(maxWidth: .infinity)
                     .cornerRadius(16)
             }
```
Comment (nuno-vieira) [inline Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift:72]: This corner radius is not need anymore:
```suggestion
            if !message.text.isEmpty {
                AttachmentTextView(message: message)
                    .frame(maxWidth: .infinity)
            }
```

File: Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift [hunk 2]
```diff
@@ -40,32 +40,38 @@ public struct VoiceRecordingContainerView<Factory: ViewFactory>: View {
...
         }
+        .padding(.all, 2)
```
Comment (nuno-vieira) [inline Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift:74]: Why is this padding needed? 🤔

**Comment by nuno-vieira** [comment]
Just added a few comments 👍 We will need to re-record the snapshot tests as well

File: Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift (hunk 2 above)
Comment (laevandus) [reply Sources/StreamChatSwiftUI/ChatChannel/MessageList/AsyncVoiceMessages/VoiceRecordingContainerView.swift:74]: There is a tiny padding needed around the VoiceRecordingView and between the bubble background. It was 4 before.

**Review by martinmitrevski — APPROVED** [review]
Looks good to me, but there are some test failures to check before merging

**Pull request merged**